test/e2e/e2e_test.go

```diff
@@ -206,8 +182,30 @@ func setup(t *testing.T, ctx *framework.TestCtx, awaitility *wait.Awaitility, us
 	_, err = awaitility.Host().WaitForUserSignup(userSignup.Name, wait.UntilUserSignupHasConditions(approvedByAdmin()...))
 	require.NoError(t, err)
 
-	return userSignup, &v1alpha1.UserAccountSpec{
-		UserID:   userSignup.Name,
+	return *userSignup
+}
+
+func postSignup(t *testing.T, route string, identity authsupport.Identity) {
+	// Call signup endpoint with a valid token.
+	emailClaim := authsupport.WithEmailClaim(uuid.NewV4().String() + "@email.tld")
+	iatClaim := authsupport.WithIATClaim(time.Now().Add(-60 * time.Second))
+	token, err := authsupport.GenerateSignedE2ETestToken(identity, emailClaim, iatClaim)
+	require.NoError(t, err)
+
+	req, err := http.NewRequest("POST", route+"/api/v1/signup", nil)
+	require.NoError(t, err)
+	req.Header.Set("Authorization", "Bearer "+token)
+	req.Header.Set("content-type", "application/json")
+	client := httpClient
```

Comment:
what's the point of this allocation?

Reply:
No point. Just didn’t do full cleanup.

Reply:
ok, no worries ;)

Reply:
Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/51/commits/3a8d5151b0f2a32f5e3cb7821ef9e2a14b9be5f2